lib/improver/wind_downscaling.py

```diff
@@ -203,10 +205,10 @@ def _setmask(self):
 
         Returns:
             (tuple) : tuple containing:
-                **hcmask** (2D array logical):
-                    True for land-points, false for Sea (HC)
-                **rcmask** (2D array logical):
-                    additionally False for invalid z_0 (RC)
+                **hcmask** (bool):
```

Comment:
I think this should probably be:
```python
    **hcmask** (np.ndarray):
        2D array of booleans etc
```

This applies to other `mask` variables within this file, where the type should be np.ndarray.